src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.*;
+
+public class Tagging {
+	
+    private Tag tag;
+    private Person person;
+    
+    public Tagging() {}
+    
+    public Tagging(Person person, Tag tag) {
```

Comment:
The constructor should be accepting a tagging operation as well. i.e. ADD or REMOVE. You can use a boolean or a enum to maintain these values.